lib/client.ml

```diff
@@ -115,6 +315,13 @@ let serve client router msg_handler =
     let%lwt message = recv_next client in
     let%lwt () = route client router message in
 
+    let%lwt () =
+      match%lwt Inbox.next !client.inbox Message.Failure_detection with
+      | Some message -> Failure_detector.handle_message client message
+      | None -> Lwt.return () in
```

Comment:
I think we should definitively work on a log framework. It feels so weird to just write `Lwt.return ()` without any explanation, even when, like in this case, this is the thing to do.

Reply:
Will be done in #10